frontend/src/components/App.tsx

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
+import {ConfigProvider, theme} from 'antd';
 import {FolderTree} from "./FolderTree";
 import {ChatHistory} from "./ChatHistory";
 import {SendChatContainer} from "./SendChatContainer";
 import {useChatContext} from '../context/ChatContext';
 import {StreamedContent} from './StreamedContent';
+import {ThemeProvider, useTheme} from '../context/ThemeContext';
+import {ThemeToggleButton} from './ThemeToggleButton'
+
+const AppContent = () => {
+    const { streamedContent, messages } = useChatContext();
+    const { isDarkMode } = useTheme();
 
-export const App = () => {
-    const {streamedContent, messages} = useChatContext();
     return (
-        <>
+        <ConfigProvider
+            theme={{
+                algorithm: isDarkMode ? theme.darkAlgorithm : theme.defaultAlgorithm,
```

Comment:
Not sure I follow the need for this prop 
```
 theme={{
                algorithm: isDarkMode ? theme.darkAlgorithm : theme.defaultAlgorithm,
```

Also if you git pull latest, index.tsx has all the providers, so I think this can be removed from here and we can update index.tsx something like below

```
<ThemeProvider>
<ChatProvider>
<FolderProvider>
                <App/>
</FolderProvider>
</ChatProvider>
</ThemeProvider>
```